db/migrate/20210302144414_drop_authorizations_table.rb

```diff
@@ -0,0 +1,14 @@
+class DropAuthorizationsTable < ActiveRecord::Migration[6.1]
+  def change
+    drop_table :authorizations do |t|
+      t.string   :provider
+      t.string   :uid
+      t.bigint  :user_id
+      t.datetime :authorized_at
```

Comment:
let's either line these all up... or not? I say just use consistent single space

```suggestion
      t.string :provider
      t.string :uid
      t.bigint :user_id
      t.datetime :authorized_at
```